ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -33,7 +31,7 @@ export default class ClientsCountsRoute extends Route {
   queryParams = {
     start_time: { refreshModel: true, replace: true },
     end_time: { refreshModel: true, replace: true },
-    ns: { refreshModel: false, replace: true },
+    ns: { refreshModel: true, replace: true },
```

Comment:
This is necessary so the `getExportCapabilities` method runs when the namespace changes.